environments/development/invoke.yml

```diff
@@ -41,8 +43,4 @@ aws:
   image_upload_bucket: "search-portal-media-uploads-dev"
   profile_name: "pol-dev"
   task_role_arn: "arn:aws:iam::322480324822:role/ecsTaskExecutionRole"
-
-# The legacy mechanism to be able to override postgres credentials during migrations
-# Keeping this around for now until we move to AWS Secrets Manager fully
-postgres:
-  credentials: null
+  bastion: "bastion.dev.surfedushare.nl"
```

Comment:
moet dit ook niet bij acceptance?

Reply:
Ja, maar dan moeten we Terraform ook uitrollen op acceptance enzo. Dus ik dacht even wachten totdat we dat hebben gedaan (misschien ook even meer dingen fixen in Terraform)

Reply:
Terraform is toch al uitgerold op acceptance? Behalve misschien het secret dat is toegevoegd 😉